### PR TITLE
Add accessibility

### PR DESCRIPTION
**Pull request by ulises2607**
### **Portfolio Accessibility Check**

- [Page titles](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R8) - Made title more descriptive

- Images:
              . [Fixed header icon](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R19) - Add anchor and alt at image
              . [Alt on hamb-icon added](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R30) - 
              . [Alt changed on class social-icon](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R62-R93)
              . [divider alt removed](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R46) - This image is just decorative and people don't need to know about the image, then it should have null alt (alt=" ").

- [Text headings](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R278) - Give a heading to the contact section

- Color Contrast - There are some contrasts to change in the buttons, the recommendation is to change the background to brown, which would not make sense because if I do it I would have to change the contrast of all the images of the template also so that the page has a harmony in terms of styles. Not changed.

- [Resize](https://github.com/ulises2607/Portfolio/pull/11/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7a) - Fixed all element positions and changed all px units to rem units

- [Interaction](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R283-R312) -  Add labels to form elements

- Moving content - No moving content issues present.

- Multimedia - No multimedia issues present

- [The basic structure of the page](https://github.com/ulises2607/Portfolio/pull/11/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R100-R136) - Deleted duplicated h1 and added h2,h3 and h4 to keep a proper hierarchy

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @ulises2607 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not met all the project requirements, kindly I suggest you make sure your linters check are passed before asking for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,343 +1,365 @@
...
 
     <section class="main">
-
-       
-        <img class="bg-main-desk" src="Images/Main/Illustration-header.png" alt="">
-        
+      <img
+        class="bg-main-desk"
+        src="Images/Main/Illustration-header.png"
+        alt=""
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] All images with the `alt` should have an alternative text which is useful for interacting with the page 👍

File: index.html [hunk 2]
```diff
@@ -1,343 +1,365 @@
...
-                    <li class="tech-btn"><a href="#">Ruby</a></li>
-                </ul>
-
-                <a href="#" class="btn-works-1">See Projects</a>
-            </div>
+      <div class="recent-w-title">
+        <h1>My Recent Works</h1>
+
+        <img src="Images/Work/Desktop/Vector.png" alt="" />
+      </div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Kindly check my comment on line `46` on the need for the `alt` to be specified with appropriate text 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @ulises2607    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with checking accessibility features✔️  
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement
- [ ] The requirement was to list all points checked and give an evaluation of either its solved, or what was done to fix it. Based on your review, you missed the following evaluations on the pull request:
- Text headings
- Color contrast
- Resize
- Interaction
- Moving content
- The basic structure of the page.

- [x] Each point on the list needs to include the result of your evaluation:
If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
- [x] If you needed to change something, [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148). Please add the links.

### Instructions:
image



_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @ulises2607    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with checking accessibility features✔️  
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement
- [ ] The requirement was to list all points checked and give an evaluation of either its solved, or what was done to fix it. Based on your review, you missed the following evaluations on the pull request:
- Text headings
- Color contrast
- Resize
- Interaction
- Moving content
- The basic structure of the page.

- [ ] Each point on the list needs to include the result of your evaluation:
If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
- [ ] If you needed to change something, [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148). Please add the links.

### Instructions:
image



_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -1,343 +1,366 @@
...
-                    <li class="tech-btn"><a href="#">CSS</a></li>
-                    <li class="tech-btn"><a href="#">HTML</a></li>
-                    <li class="tech-btn"><a href="#">Bootstrap</a></li>
-                    <li class="tech-btn"><a href="#">Ruby</a></li>
-                </ul>
-
-                <a href="#" class="btn-works-1">See Projects</a>
-            </div>
+      <div class="recent-w-title">
+        <h1>My Recent Works</h1>
```
Comment (shahadat3669) [inline index.html]: Great work so far! 👏  It appears that you're using the `<h1>` tag for each section title. However, it's recommended to have only one `<h1>` tag per webpage for better accessibility. Consider replacing the `<h1>` tag with an appropriate heading tag, such as `<h2>` to `<h6>`, depending on the hierarchy and relevance of the titles within your code. You can refer to the [documentation on heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns) for more information.

By implementing these suggestions, you'll ensure that your design is consistent and adheres to best practices. Keep up the excellent work! Happy coding! 😊

**_Note that most of the headings in your webpage are not structured well. please consider the same requirement for all headings._**

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ Good PR Title
✅ Good PR Description
✅ Good use of GitHub Flows

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi **Team**, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**